pkg/plugins/plugins.go

```diff
@@ -50,6 +50,12 @@ func RegisterSchedulerPlugin(plugin interface{}) {
 		plugins.reconcilePlugin = t
 		registered = true
 	}
+	if t, ok := plugin.(ContainerSchedulingStateUpdater); ok {
+		log.Logger().Debug("register scheduler plugin",
+			zap.String("type", "ContainerSchedulingStateUpdater"))
```

Comment:
Yes. simplified and updated.

Reply:
But we could not move to `switch` because the logic here allows an object that implements multiple plugin interfaces. I added a UT for this too in the latest patch.